config.json

```diff
@@ -920,6 +936,142 @@
         "nugetId": "Xamarin.Firebase.ProtoliteWellKnownTypes",
         "dependencyOnly": false
       },
+      {
+        "groupId": "com.google.android.play",
+        "artifactId": "core",
+        "version": "1.9.0",
+        "nugetVersion": "1.9.0",
+        "nugetId": "Xamarin.Google.Android.Play.Core",
+        "dependencyOnly": false
+      },
+      {
+        "groupId": "com.google.mlkit",
+        "artifactId": "common",
+        "version": "16.0.0",
+        "nugetVersion": "16.0.0",
```

Comment:
I wonder if we should call these `116.0.0` for consistency.  I realize it's not needed because we never shipped a `16.X` previously, but it might reduce confusion of why some of our Firebase packages are `116.X` and some are `16.X`.

Reply:
Makes sense.